audbcards/core/datacard.py

```diff
@@ -42,7 +42,6 @@ def _render_template(self):
         # Provide Jinja filter access to Python build-ins/functions
         environment.filters.update(
```

Comment:
Is the zip filter used at all any longer?

If not, one could discard the entire  `environment.filters.update()` statement.

Reply:
Thanks for the suggestion, I forgot that we no longer need `zip` to create the tables. I removed the `environment.filters.update()` statement.